### PR TITLE
Update to pretty demo 🎉

### PR DESCRIPTION
**Pull request by abdonrd**
Before:

screen shot 2015-12-14 at 12 34 19

After:

screen shot 2015-12-14 at 12 34 31

### CODE REVIEW

File: demo/index.html [hunk 1]
```diff
@@ -16,89 +16,68 @@
   <meta name="apple-mobile-web-app-capable" content="yes">
 
   <script src="../../webcomponentsjs/webcomponents-lite.js"></script>
+
+  <link rel="import" href="../../iron-demo-helpers/demo-snippet.html">
+  <link rel="import" href="../../iron-demo-helpers/demo-pages-shared-styles.html">
   <link rel="import" href="../../paper-styles/demo-pages.html">
```
Comment (notwaldorf) [inline demo/index.html]: you can remove `demo-pages`

File: demo/index.html [hunk 2]
```diff
@@ -16,89 +16,68 @@
...
-      --paper-radio-button-unchecked-color: var(--paper-red-900);
-      --paper-radio-button-unchecked-ink-color: var(--paper-red-900);
-    }
-
-    paper-radio-button.green {
-      --paper-radio-button-checked-color: var(--paper-green-500);
-      --paper-radio-button-checked-ink-color: var(--paper-green-500);
-      --paper-radio-button-unchecked-color: var(--paper-green-900);
-      --paper-radio-button-unchecked-ink-color: var(--paper-green-900);
+    demo-snippet {
```
Comment (notwaldorf) [inline demo/index.html]: i think this is already in the `demo-pages-shared-styles`

File: demo/index.html [hunk 3]
```diff
@@ -16,89 +16,68 @@
...
-        <paper-radio-button checked>Hydrogen</paper-radio-button>
-        <paper-radio-button checked>Nitrogen</paper-radio-button>
-        <paper-radio-button checked>Calcium</paper-radio-button>
-      </div>
-    </div>
+  <div class="vertical-section-container centered">
+    <h3>Radio buttons can be checked or unchecked, or disabled entirely</h3>
+    <demo-snippet class="centered-demo">
+      <template>
+        <paper-radio-button>Radio button</paper-radio-button>
```
Comment (notwaldorf) [inline demo/index.html]: I would just label this "Radio"

**Comment by notwaldorf** [comment]
Looking good! Just a couple of comments

File: demo/index.html (hunk 1 above)
Comment (abdonrd) [inline demo/index.html]: 👌

File: demo/index.html (hunk 2 above)
Comment (abdonrd) [inline demo/index.html]: 👌

File: demo/index.html (hunk 3 above)
Comment (abdonrd) [inline demo/index.html]: Okey!

**Comment by abdonrd** [comment]
Done!

**Comment by notwaldorf** [comment]
Awesome! Thanks so much for helping out!

**Comment by abdonrd** [comment]
Your welcome @notwaldorf! :)

**Pull request merged**